docs/reference/installation.rst

```diff
@@ -277,38 +241,19 @@ Then, add a new custom firewall handlers for the admin:
                 pattern:  ^/(_(profiler|wdt)|css|images|js)/
                 security: false
 
-            # -> custom firewall for the admin area of the URL
+            # Firewall for the admin area of the URL
             admin:
-                pattern:            /admin(.*)
-                context:            user
+                anonymous: true
+                pattern: /admin(.*)
+                provider: sonata_user_bundle
```

Comment:
```suggestion
```
IMO we should use [form_login Authentication Provider](https://symfony.com/doc/4.4/security/form_login.html)

Reply:
We are already using the form login authentication provider. Check the below code, and the implementation.

As for your comment to remove that provider line. It cannot be removed. It helps to configure what user provider we want to use. It is needed to do things like modify the query to allow users to be disables/enabled.